src/web/mjs/connectors/BilibiliComics.mjs

```diff
@@ -13,7 +13,7 @@ export default class BilibiliComics extends BilibiliManhua {
         };
         //override BilibiliManhua config as we need lanuage switch. Difference is handled in the main code with (this.config.language) ? : test
         this.config = {
-            quality:  {
+            format:  {
```

Comment:
This new property seems not to be used in the committed changes, is it really required?

Reply:
OK just noticed it is for the base class ...